src/main/java/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+public class Rectangle {
+    private int width;
+    private int height;
+
+    // "strange" constructor with no required properties, which seems to be needed for the task
+    public Rectangle() {
```

Comment:
You can remove this constructor - there is no need in creating such object without properties, because there aren't any getter() methods for this class (for current case it works because main is in the same class, but it will not work for some real project if we use this object outside object's class)

Reply:
Done.  Redundant constructor removed.